packages/reputation-miner/Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM node:14-bullseye
+FROM node:20.11.0-buster
 RUN apt-get update || : && apt-get install python -y
 COPY ./packages ./packages
 COPY ./package.json ./
 COPY ./package-lock.json ./
 COPY ./artifacts ./artifacts
 RUN git config --global url."https://".insteadOf "ssh://git@"
 RUN npm ci
-RUN cd ./packages/reputation-miner/ && npm ci
```

Comment:
Why is this no longer necessary?

Reply:
In the same way that the `package-lock` files in the packages were unnecessary ( 158e1464145844ed73dd1a88c653d655ca8a130b), there's no point in running `npm ci` inside the package directories separately.